install_config/storage_examples/ceph_rbd_dynamic_example.adoc

```diff
@@ -0,0 +1,327 @@
+====
+
+[[ceph-rbd-dynamic-example-installing-the-ceph-common-package]]
+== Installing the ceph-common Package
+
+The *ceph-common* library must be installed on *all schedulable* {product-title} nodes:
+
+[NOTE]
+====
+The {product-title} all-in-one host is not often used to run pod workloads and,
+thus, is not included as a schedulable node.
+====
+
+----
+# yum install -y ceph-common
+----
+
+[[ceph-rbd-dynamic-example-create-pool-for-dynamic-volumes]]
+== Create pool for dynamic volumes
+
+It is recommended to create a pool for your dynamic volumes to live.
+
+[NOTE]
+====
+Using the default pool of rbd is an option, but not recommended. 
+====
+
+From a Ceph admin or Mon node, create a new pool for dynamic volumes
+----
+ceph osd pool create kube 1024
+ceph auth get-or-create client.kube mon 'allow r' osd 'allow class-read object_prefix rbd_children, allow rwx pool=kube' -o ceph.client.kube.keyring
+----
+
+[[ceph-rbd-dynamic-example-creating-the-ceph-secret]]
+== Creating the Ceph Secret
+
+The `ceph auth get-key` command is run on a Ceph *MON* node to display the key
+value for the *client.admin* user:
+
+.Ceph Secret Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-secret
+  namespace: kube-system
+data:
+  key: QVFBOFF2SlZheUJQRVJBQWgvS2cwT1laQUhPQno3akZwekxxdGc9PQ== <1>
+type:
+  kubernetes.io/rbd <2>
+
+----
+<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph auth get-key client.admin | base64` command, then copying the output and pasting it as the secret key's value.
+<2> This is required for Ceph RBD to work with dynamic provisioning.
+====
+
+Save the secret definition to a file, for example *_ceph-secret.yaml_*,
+then create the secret:
+
+====
+----
+$ oc create -f ceph-secret.yaml
+secret "ceph-secret" created
+----
+====
+
+Verify that the secret was created:
+
+====
+----
+# oc get secret ceph-secret
+NAME          TYPE                DATA      AGE
+ceph-secret   kubernetes.io/rbd   1         5d
+----
+====
+
+[[ceph-rbd-dynamic-example-creating-the-ceph-user-secret]]
+== Creating the Ceph User Secret
+
+The `ceph auth get-key` command is run on a Ceph *MON* node to display the key
+value for the *client.kube* user:
+
+.Ceph Secret Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-user-secret
+  namespace: default
+data:
+  key: QVFCbEV4OVpmaGJtQ0JBQW55d2Z0NHZtcS96cE42SW1JVUQvekE9PQ== <1>
+type:
+  kubernetes.io/rbd <2>
+
+----
+<1> This base64 key is generated on one of the Ceph MON nodes using the `ceph auth get-key client.kube | base64` command, then copying the output and pasting it as the secret key's value.
+<2> This is required for Ceph RBD to work with dynamic provisioning.
+====
+
+Save the secret definition to a file, for example *_ceph-user-secret.yaml_*,
+then create the secret:
+
+====
+----
+$ oc create -f ceph-user-secret.yaml
+secret "ceph-user-secret" created
+----
+====
+
```

Comment:
Same here:

    .Verify that the secret was created:
    +
    ----
    # oc get secret ceph-user-secret
    NAME               TYPE                DATA      AGE
    ceph-user-secret   kubernetes.io/rbd   1         5d
    ----